astropy/timeseries/sampled.py

```diff
@@ -150,28 +150,47 @@ def fold(self, period=None, midpoint_epoch=None):
         ----------
         period : `~astropy.units.Quantity`
             The period to use for folding
-        midpoint_epoch : `~astropy.time.Time`
+        epoch : `~astropy.time.Time`
             The time to use as the midpoint epoch, at which the relative
-            time offset will be 0. Defaults to the first time in the time
-            series.
+            time offset / phase will be 0. Defaults to the first time in the
+            time series.
+        normalize_phase : bool
+            If False phase is returned as `~astropy.time.TimeDelta`, otherwise
+            as dimensionless `~astropy.units.Quantity`.
+        phase_range : float
+            Length of phase to return, centered on ``epoch``. Default is 1 to
+            return the full phase.
```

Comment:
Can't this be done by users after getting the folded time series back? Why do we need to special case this as a kwarg? (I'd suggest removing this)

Reply:
At least I think this one will be harder to agree on before feature freeze so I'm suggesting to remove for now.

Reply:
I'm still not clear on why this needs to be an option here and why this can't just be done afterwards

Reply:
Also what happens if ``phase_length=5``?